browser-extension/common/src/popup/components/PopupTabs/index.tsx

```diff
@@ -5,6 +5,13 @@ import PinnedRecords from "../PinnedRecords";
 import RecentRecords from "../RecentRecords";
 import { PushpinFilled, CheckCircleOutlined, ClockCircleOutlined } from "@ant-design/icons";
 import "./popupTabs.css";
+import { EVENT, sendEvent } from "../../events";
+
+const tabKeyNameMap: Record<string, string> = {
+  "0": "pinned_rules",
+  "1": "recently_used",
+  "2": "executed_rules",
+};
```

Comment:
Instead of using index as tab key, use string version.
```js
enum PopupTabKey {
  PINNED_RULES = "pinned_rules",
  RECENTLY_USED = "recently_used",
  EXECUTED_RULES = "executed_rules",
}
```